lib/fog/compute/google/models/instance_group.rb

```diff
@@ -27,6 +27,25 @@ def destroy(_async = true)
           data = service.delete_instance_group(name, zone_name)
         end
 
+        def add_instance(instance_name)
+          requires :identity, :zone
+
+          service.add_instance_group_instance(identity, zone_name, instance_name)
+        end
+
+        def remove_instance(instance_name)
+          requires :identity, :zone
+
+          service.remove_instance_group_instance(identity, zone_name, instance_name)
+        end
+
+        def list_instances
```

Comment:
Additionally, if there's no data it passes back `nil` - is that ok or should it be empty array?